README.md

```diff
@@ -1,101 +1,114 @@
 # Boundary
-![](boundary.png)
 
-----
+- What is Boundary: https://developer.hashicorp.com/boundary/docs/overview/what-is-boundary
+- Website: https://www.boundaryproject.io/
+- Forums: [HashiCorp Discuss](https://discuss.hashicorp.com/c/boundary/)
+- Documentation: [https://boundaryproject.io/docs](https://boundaryproject.io/docs)
+- Tutorials: [HashiCorp's Learn Platform](https://developer.hashicorp.com/boundary/tutorials)
 
-**Please note**: We take Boundary's security and our users' trust very
-seriously. If you believe you have found a security issue in Boundary,
-_please responsibly disclose_ by contacting us at
-[security@hashicorp.com](mailto:security@hashicorp.com).
+<img alt="Boundary" src="boundary.png" alt="Image" width="500px"/>
 
-----
+Boundary provides a simple and secure way to access hosts and critical systems without the need to handle credentials, set up firewalls, or expose your network.
+
+With Boundary you can:
 
-Boundary provides simple and secure access to hosts and services.
+* Enable single sign-on to target services and applications via external identity providers
+* Provide Just-in-Time network access to resources, wherever they reside 
+* Enable passwordless access to machines with dynamic credentials via [HashiCorp Vault](https://www.vaultproject.io/)
+* Automate discovery of new target systems
+* Record and manage privileged sessions
+* Standardize your team's access workflow with a consistent experience for any type of infrastructure across any provider
 
-Traditional approaches like SSH bastion hosts or VPNs require distributing and managing 
-credentials, configuring network controls like firewalls, and exposing the private 
-network. Boundary provides a secure way to access hosts and critical systems without 
-having to manage credentials or expose your network, and is entirely open source.
 
 Boundary is designed to be straightforward to understand, highly scalable, and 
 resilient. It can run in clouds, on-prem, secure enclaves and more, and does not require
 an agent to be installed on every end host.
 
-Unlike firewalls, Boundary performs per-access authentication and
-authorization checks, allowing for much higher level mappings of users to
-services or hosts than at network layers. Although complementary to secrets
-managers (like HashiCorp's own [Vault](https://www.vaultproject.io/)),
-Boundary fills a different niche, allowing the credential that is eventually
-used to be hidden entirely from the user.
+
+<a href="https://www.youtube.com/watch?v=DCkDqZdATC0">
+  <img src="https://www.boundaryproject.io/_next/image?url=https%3A%2F%2Fwww.datocms-assets.com%2F58478%2F1664218843-boundary-illustration-option2-1.png&w=3840&q=75" alt="Watch the video" width="320" height="180">
+</a>
```

Comment:
Was this image supposed to link to Van's video? I didn't realize it was a video at first, until I hovered over the graphic. Also, I think the video is an old version. If you go to 5:00 Van uses the old product name. This should be the link to the new video:

https://www.datocms-assets.com/2885/1694719896-how-boundary-works-v3.mp4

To verify, if you skip to 4:15, Van uses the new product name.